index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Mobile First</title>
+</head>
+
+<body>
+    <main id="main-section">
+
+        <nav class="navbar">
+            <a href="#">Nasir</a>
+            <img src="images/Union.png" alt="button">
+        </nav>
```

Comment:
- Great Job for using the `nav` button for your hamburger 👍, However, the `nav` is not part of the main content so make it outside the `main` tag and wrap it with the `header` tag [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. 
- I think it would be better if you make your logo outside the `nav` tag because it is not part of the navigation 👌
check the bellow format👇
```
<header>
   <a>....</a>
   <nav>....</nav>
</header>
<main>
  ....
```